api/src/controllers/record-controller.js

```diff
@@ -44,7 +44,7 @@ let EditManagementPlan = require('./put/management-plan');
  * @param {*} res
  * @param {*} next
  */
-exports.protectedOptions = function(args, res, next) {
+exports.protectedOptions = function (args, res, next) {
```

Comment:
I noticed elsewhere that the old lint issue of `{}` getting changed to `{ }` was fixed, so thats good.

But looks like there is still one last setting in your local vscode that is adding spaces between `function` and the brackets.

Reply:
Its this setting, I believe: `insertSpaceBeforeFunctionParenthesis`

It defaults to false, so somewhere it must have gotten toggled to true.

Reply:
I run lint-fix before every commit. This has nothing to do with my VS Code settings. So lint-fix is not catching this.

Reply:
When I run lint-fix locally, it removes those spaces.  The only reason it wouldn't, is if the config file was overwritten by another setting.